src/python_pybind11/src/_gz_math_pybind11.cc

```diff
@@ -78,14 +79,11 @@ PYBIND11_MODULE(BINDINGS_MODULE_NAME, m)
 
   gz::math::python::defineMathColor(m, "Color");
 
-  gz::math::python::defineMathDiffDriveOdometry(
-    m, "DiffDriveOdometry");
+  gz::math::python::defineMathDiffDriveOdometry(m, "DiffDriveOdometry");
```

Comment:
are these change required ? I would prefer to keep only the changes related with the PR title, You can open a PR cleanup the style